jcl/src/java.base/share/classes/java/lang/invoke/VarHandle.java

```diff
@@ -343,6 +343,7 @@ MethodType accessModeType(Class<?> receiver, Class<?> type, Class<?>... args) {
 /*[IF Java14]*/
 	static final BiFunction<String, List<Integer>, ArrayIndexOutOfBoundsException> AIOOBE_SUPPLIER = null;
 	VarForm vform = null;
+	final boolean exact;
```

Comment:
Is this need for 14+ or just 16+?

Reply:
The addition of the `exact` field impacts a constructor [`VarHandle(VarForm varForm)`] which was introduced in Java 14. In order to avoid duplication of the constructor (100+ lines of code), the field is surrounded using the `Java14` flag.

Reply:
Now, the all changes are only exposed in `Java16`.